main.go

```diff
@@ -36,16 +36,16 @@ type link struct {
 
 // D3 is the structure used to construct the data for the frontend D3 graph.
 type D3 struct {
-	Nodes []node `json:"nodes"`
-	Links []link `json:"links"`
+	Nodes      []node `json:"nodes"`
+	Links      []link `json:"links"`
+	LastUpdate string `json:"lastUpdate"`
```

Comment:
As said in a previous comment the D3 data will be subject to change in the near future, and it should only contain proper graph data. `Dashboard` not only contains RL details, but every other kind of not graph-related information.